module.mk

```diff
@@ -134,6 +134,7 @@ LIBS_TO_BUILD =
 ifdef TT_METAL_ENV_IS_DEV
 LIBS_TO_BUILD += \
 	python_env/dev \
+	python_env/dev/editable \
```

Comment:
For some reason, I can't get the additional target to work here:
If I add `python_env/dev/stubs`, I get an error with the execution of  `python_env/dev/stubs` that looks like this:
```
Generating stubs...
bash: stubgen: command not found
make: *** [/home/ubuntu/actions-runner/_work/tt-metal/tt-metal/tt_metal/python_env/module.mk:45: /home/ubuntu/actions-runner/_work/tt-metal/tt-metal/build/python_env/.installed-stubs] Error 127
```

https://github.com/tenstorrent-metal/tt-metal/actions/runs/8572730164/job/23495940653